### DISCUSSION TITLE
Cutting releases to include bug fixes

### DISCUSSION THREAD

**Discussion by grantseltzer** [aquasecurity/tracee#642] [Development]
In #639 the reporter mentions an issue they have with 0.5.0 because of a bug that's been fixed since we cut the release. I think we should be cutting minor releases to include bug fixes, would it make sense to do this reactively if it's a particularly bad bug but otherwise have a weekly/bi-weekly minor release schedule?

**Comment by itaysk** [comment]
since our release process is chronological (we don't cherry-pick commits), we we have to consider not just bug fixes, but also the features that will end up in the release - a release should to be stable in terms of the user experience. a current example, we can't cut a release before the gotemplate stuff is complete (it's actually at the finish line already, will likely release 0.5.1 in coming days).
I guess what I'm saying is that the release schedule is driven by functionality, and bug fixes are included in the functional release if they happen to be merged in time. If we get to a situation where we don't have any new functionality to release in a some time, we can cut a release just for fixes of course, but that's a situation that hopefully will be rare.
